### PR TITLE
:zap: Replace `archiver` with `zip-lib`

### LINKED ISSUE #275: `neu build` hangs up when run with bun
[Bun](https://bun.sh/) is a fast node.js alternative that is mostly compatible with its API. The "mostly" part results in `archiver` package not completing its zipping when run with bun, hanging forever. This prevents building neutralino apps if Bun is your node.js package runner, or if you use the neutralino-cli bundler programmatically through `const {bundleApp} = require('@neutralinojs/neu/src/modules/bundler.js');` in a bun application. (Which is my case.)

изображение
No messages appear after "Making app bundle ZIP file".

Now, I have zipping/unzipping capabilities in my Bun extension already, and I also faced problems with Archiver and used [zip-lib](https://www.npmjs.com/package/zip-lib) instead. It's more modern, too.

I will probably make a PR with the replacement package for this issue, as its use case is exotic enough to not care about.

### PR DESCRIPTION
**Pull request by CosmoMyzrailGorynych**
Closes #275.
TL;DR: Allow using `neu build` with [Bun](https://bun.sh/) as node.js package runner.

Observed side effects: 

1. Zipping an empty neutralino app (made by `neu create sausage`) is roughly twice as faster. Good!

<details> <summary>Before/After</summary>
Before:<br/>
<br/><br/>
After:<br/>

</details>

2. The zip file is a _smidgen_ larger. (5 230 417 --> 5 247 436 bytes.) This is due to `zip-lib` not having any compression options  you can change.

### CODE REVIEW

**Review by shalithasuranga — APPROVED** [review]
Thanks so much for sending this pull-request @CosmoMyzrailGorynych The `zip-lib` module looks great :tada:
